### ISSUE TITLE
cant run it on mac os sierra. [ 10.12. 3 ]

### ISSUE THREAD

**Issue by n0bzeR** [iina/iina#5141]
can add support old mac os , for new iina ????
 im using sierra mac os, not high sierra (((
why new builds not work on 10.12.?
 why need 10.3.x ??

**Comment by low-batt** [comment]
This is discussed in issue #4634. The problem with supporting macOS Sierra 10.12 started with this note from the [Xcode 14 Release Notes](https://developer.apple.com/documentation/xcode-release-notes/xcode-14-release-notes):
> Building for deployment to OS releases older than macOS 10.13, iOS 11, tvOS 11, and watchOS 4 is no longer supported. (92834476)

The key problem with Apple having dropped support is that projects IINA is dependent upon will follow Apple's lead and also drop support. IINA made a concerted effort to remain compatible with Xcode 13 in order to continue to support 10.11 and 10.12. That effort failed due to trouble compiling the libraries from other projects that IINA is dependent upon. So IINA 1.3.5 requires macOS High Sierra 10.13 or later.

And it gets worse. See issue #4837. The current development version of IINA requires macOS Catalina 10.15. Again this requirement was imposed upon IINA by the requirements of the libraries IINA is built upon. This will be a bummer for users trapped on macOS Mojave 10.14 due to the need to run an abandonware app that was never upgraded to 64-bit. But IINA has no choice but to comply with the requirements of the included libraries.

**Comment by n0bzeR** [comment]
> This is discussed in issue #4634. The problem with supporting macOS Sierra 10.12 started with this note from the [Xcode 14 Release Notes](https://developer.apple.com/documentation/xcode-release-notes/xcode-14-release-notes):
> 
> > Building for deployment to OS releases older than macOS 10.13, iOS 11, tvOS 11, and watchOS 4 is no longer supported. (92834476)
> 
> The key problem with Apple having dropped support is that projects IINA is dependent upon will follow Apple's lead and also drop support. IINA made a concerted effort to remain compatible with Xcode 13 in order to continue to support 10.11 and 10.12. That effort failed due to trouble compiling the libraries from other projects that IINA is dependent upon. So IINA 1.3.5 requires macOS High Sierra 10.13 or later.
> 
> And it gets worse. See issue #4837. The current development version of IINA requires macOS Catalina 10.15. Again this requirement was imposed upon IINA by the requirements of the libraries IINA is built upon. This will be a bummer for users trapped on macOS Mojave 10.14 due to the need to run an abandonware app that was never upgraded to 64-bit. But IINA has no choice but to comply with the requirements of the included libraries.


.. okay
its sad...
thanks for reply

**Comment by low-batt** [comment]
Yes, very sad. I too have old Macs running old versions of macOS. The IINA team really did not want to drop support. But we have to adhere to the requirements of the 3<sup>rd</sup> party libraries that IINA is built on top of.

**Comment by anohren** [comment]
@low-batt Thanks for the nice explanation. As a regular user of Mojave it's a bummer, but the old version still works well enough there (the skin tones are a bit orangey though...?)

In the end it's my own choice not to boot into Monterey with OCLP (which works well enough).
